ui/keyboardMove/src/plugins/keyboardMove.ts

```diff
@@ -120,6 +124,19 @@ export default (opts: Opts) => {
   };
 };
 
+function iccfToUci(v: string) {
+  if (v.match(/^5(1|8)7(1|8)$/)) return 'o-o';
```

Comment:
What if you are moving for instance a rook from e1 to g1? For this reason, this kind of regex won't work here, but luckily we can just convert this to `e1g1`, which is valid UCI for castling.